src/journal-remote/journal-remote-main.c

```diff
@@ -1136,6 +1145,16 @@ static int run(int argc, char **argv) {
                 s.check_trust = !arg_trust_all;
         }
 
+        memset(&s.metrics, 0xFF, sizeof(s.metrics));
+        if (arg_max_use)
+                s.metrics.max_use = arg_max_use;
+        if (arg_max_size)
+                s.metrics.max_size = arg_max_size;
+        if (arg_keep_free)
+                s.metrics.max_size = arg_keep_free;
+        if (arg_n_max_files)
+                s.metrics.n_max_files = arg_n_max_files;
```

Comment:
Then, you can simply use `journal_reset_metrics()` and copy the values.

Reply:
done